docs/getting-started.md

```diff
@@ -192,7 +188,10 @@ To do this we will simply call `Kill` on the task after waiting a couple of seco
 	}
 
 	status := <-exitStatusC
-	fmt.Printf("redis-server exited with status: %d\n", status)
+	if status.Err != nil {
```

Comment:
Looking at the code changes and now that we have three fields on this struct, maybe its better if we un-export the fields and add a method for it.  The calling code would look like this:

```go

exitResult, err := task.Wait(ctx)
if err != nil {
    return err
}

code, timestamp, err := exitResult.Result()
if err != nil {
    return err
}
```

This way, it makes the consumer handle the error.  Before, I feel like its too easy to skip checking `status.Err` and people would forget to do that.